packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -440,23 +454,29 @@ class TreeTableState<T extends TreeNode<T>> extends State<TreeTable<T>>
     // TODO(gmoothart): Add expand/collapse with left/right keys
   }
 
-  void _moveSelection(LogicalKeyboardKey key, ScrollController scrollController,
-      double viewportHeight) {
+  void _moveSelection(
+    LogicalKeyboardKey key,
+    ScrollController scrollController,
+    double viewportHeight,
+  ) {
     // get the index of the first item fully visible in the viewport
     final firstItemIndex =
         (scrollController.offset / _Table.defaultRowHeight).ceil();
 
     final minCompleteItemsInView =
         (viewportHeight / _Table.defaultRowHeight).floor() - 1;
     final lastItemIndex = firstItemIndex + minCompleteItemsInView - 1;
-    var newSelectedNodeIndex = 0; // ??
+    int newSelectedNodeIndex;
 
     if (key == LogicalKeyboardKey.arrowDown) {
       // move selection down if there is a node lower down.
       newSelectedNodeIndex = min(selectedNodeIndex + 1, items.length - 1);
     } else if (key == LogicalKeyboardKey.arrowUp) {
       // move selection up if there is a node lower down.
       newSelectedNodeIndex = max(selectedNodeIndex - 1, 0);
+    } else {
+      // Nowhere to move
```

Comment:
add a todo to move selection on left / right keys when we cannot collapse or expand in the proper direction. For example, pressing the left arrow on a collapsed node should select the collapsed node's parent.